Volume/Views/MainView/MainView.swift

```diff
@@ -61,6 +62,11 @@ struct MainView: View {
                 selectedTab = .home
             }
         }
+        .onChange(of: notifications.openedWeeklyDebrief) { newValue in
```

Comment:
`newValue` is ambiguous. I actually have no idea what it represents. Can you pick a more descriptive name?